README.md

```diff
@@ -35,14 +36,22 @@ Podane wartości można również wprowadzić w panelu zakładce *Konfiguracja -
 
 ## Dodatkowe informacje
 ### Obsługa płatności po stronie klienta
-Wpłaty które powstają po wykonaniu płatności BillTech, to tzw. opłaty tymczasowe. Są tworzone aby użytkownik widział wykonaną opłatę w userpanelu. Wpłaty tymczasowe również umożliwiają natychmiastowe odblokowanie usług w przypadku blokady z powodu niepłacenia. Opłaty tymczasowe przestają być potrzebne w momencie pojawienia się opłat z banku, wtedy mogą zostać zamknięte, po czym przestają być widoczne w panelu admina. Istnieją 3 możliwości ich zamykania:
+Opłaty które powstają po wykonaniu płatności BillTech, to tzw. opłaty tymczasowe. Są tworzone aby użytkownik oraz administrator systemu widzieli wykonaną płatność. Opłaty tymczasowe umożliwiają natychmiastowe odblokowanie usług w przypadku blokady z powodu niepłacenia. 
```

Comment:
W LMS to się chyba nazywa operacja kasowa, a my możemy nazwać to wpłatą tymczasową. Czyli:
**Operacje kasowe**, które powstają... to tzw. **wpłaty tymczasowe**...